Tests/Foundation/HTTPCookieStorageMonitorTests.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  HTTPCookiesStorageMonitorTests.swift
+//  XestiMonitors
+//
+//  Created by Angie Mugo on 2018-05-15.
+//
+//  © 2018 J. G. Pusey (see LICENSE.md)
+//
+
+import Foundation
+import XCTest
+@testable import XestiMonitors
+
+internal class HTTPCookiesStorageMonitorTests: XCTestCase {
+    let notificationCenter = MockNotificationCenter()
+    let httpCookieStorage = HTTPCookieStorage()
```

Comment:
Let's go with `cookieStorage` here too. (And swap lines 15 and 16 -- I like alphabetical order.)